src/class-elements/grammar-special-prototype-accessor-meth-valid.case

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Accessor Methods can be named "prototype"
+info: |
+    Class Definitions / Static Semantics: Early Errors
+
+    ClassElement : MethodDefinition
+        It is a Syntax Error if PropName of MethodDefinition is not "constructor" and HasDirectSuper of MethodDefinition is true.
+        It is a Syntax Error if PropName of MethodDefinition is "constructor" and SpecialMethod of MethodDefinition is true.
```

Comment:
The description is a bit loose, I'd make a better use of the ClassDefinitionEvaluation

```
Runtime Semantics: ClassDefinitionEvaluation

 ClassTail : ClassHeritage_opt { ClassBody_opt } 

...
6. Let proto be OrdinaryObjectCreate(protoParent).
...
14. Perform MakeConstructor(F, false, proto).
...
20. For each ClassElement m in order from methods, do
  a. If IsStatic of m is false, then
    i. Let status be PropertyDefinitionEvaluation of m with arguments proto and false.
```

and followed by PropertyDefinitionEvaluation

```
PropertyDefinitionEvaluation

MethodDefinition : PropertyName ( UniqueFormalParameters ) { FunctionBody }

    ...
    3. Let desc be the PropertyDescriptor { [[Value]]: methodDef.[[Closure]], [[Writable]]: true, [[Enumerable]]: enumerable, [[Configurable]]: true }.
    4. Return ? DefinePropertyOrThrow(object, methodDef.[[Key]], desc).

MethodDefinition : get PropertyName () { FunctionBody }

    ...
    9. Let desc be the PropertyDescriptor { [[Get]]: closure, [[Enumerable]]: enumerable, [[Configurable]]: true }.
    10. Return ? DefinePropertyOrThrow(object, propKey, desc).

MethodDefinition : set PropertyName ( PropertySetParameterList ) { FunctionBody }

    ...
    8. Let desc be the PropertyDescriptor { [[Set]]: closure, [[Enumerable]]: enumerable, [[Configurable]]: true }.
    9. Return ? DefinePropertyOrThrow(object, propKey, desc).
```